test/t8_gtest_custom_assertion.hxx

```diff
@@ -73,6 +73,9 @@ element_equality (const char *ts_expr, const char *tree_class_expr, const char *
 #define EXPECT_ELEM_EQ(scheme, tree_class, elem1, elem2) \
   EXPECT_PRED_FORMAT4 (element_equality, (scheme), (tree_class), (elem1), (elem2))
 
+#define ASSERT_ELEM_EQ(scheme, tree_class, elem1, elem2) \
+  ASSERT_PRED_FORMAT4 (element_equality, (scheme), (tree_class), (elem1), (elem2))
+
```

Comment:
Code duplication from the lines above
```suggestion
```

Reply:
The lines above are not the same. 

The one is an assertion, the other one is an expectation.